README.md

```diff
@@ -2,6 +2,34 @@
 
 ## Running locally
 
+Before running the application locally in the form of a Docker container, any already running instances of the application (also in the form of a Docker container) should be stopped to avoid port conflicts, unless
+other ports have been assigned to each application instance. 
+
+To run the application in the form of a Docker container, run the command `docker compose up -d` from the root of the application directory.
+
+The database engine containing the `timecard schema` should be running already.
+
+#### Flow
+
+First, Docker Compose will run the container containing Liquibase, which will perform any necessary DB migrations. This container will be killed automatically. The container logs contain a message about the success/failure of the migration execution.
```

Comment:
I would describe the container as terminating with a zero exit code i.e. it doesn't get killed it just completes successfully. If it fails it will return a non zero exit code. However, as you correctly point out, it is always the logs that contain the truth.

Reply:
Txt changed